### ISSUE TITLE
FoA Moddb suggestion

### ISSUE THREAD

**Issue by Rigmored** [Middle-Earth-Project-Team/mod#129]
Hey, would just like to say this is a magnificent job you have done. Might I offer a few suggestions though?

At the first scenario the culture of the majority of Arnor is Arnorian whilst its kings are Dunedain. I believe this should be corrected by either making the kings Arnorian or changing the culture of the provinces. It would be more acceptable if these provinces and kings changed their culture to Dunedain at the time period of the second scenario due to scattering of Arnor's residents by the great plague and Angmars' attacks.

Also, the addition of one or two Black Numernorean provinces in Angmar would help with the story since they were apparently strong soldiers for them.

**Comment by Lonhaldar** [comment]
I agree to add some BN provinces in Angmar during the reign of the Witch-King!
For Dunedain/Arnorian in Arnor : if we change that, we'll need to rework the bonuses of the two cultures i think, no?
